spec/requests/sessions/sign_in_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sign up', type: :request do
+  let(:sign_in_url) { '/auth/sign_in' }
+  let(:user) { create(:user) }
+
+  subject { post sign_in_url, params: sign_in_params }
+
+  context 'when sign in params are valid' do
+    let(:sign_in_params) do
+      {
+        user:
+          {
+            email: user.email,
+            password: user.password
+          }
+      }
+    end
+
+    context 'when user has not confirmed the account' do
+      before { subject }
+
+      it 'returns unauthorized status' do
+        expect(response).to have_http_status(:unauthorized)
+      end
+
+      it 'returns confirmation account message' do
+        expect(errors.first).to include I18n.t('api.errors.confirmation_email')
+      end
+    end
+
+    context 'when user has not confirmed the account' do
+      before do
+        user.confirm
```

Comment:
Is the confirm feature tested? Just asking

Reply:
There is another [ticket](https://trello.com/c/VJUN2VuO/16-3-as-a-user-i-should-be-able-to-confirm-my-account-after-signing-up-in-order-to-ensure-that-all-the-users-email-are-valid) for that!

Reply:
Well, then that feature shouldn't be enabled yet 🤔